src/morphological_operations.jl

```diff
@@ -99,3 +99,90 @@ which is defined as the arithmetic difference between the internal and the exter
 `region` allows you to control the dimensions over which this operation is performed.
 """
 morpholaplace(img::AbstractArray, region=coords_spatial(img)) = dilate(img, region) + erode(img, region) - 2img
+
+
+struct FalsePadded
```

Comment:
By reading the paper, I learned that the algorithm only uses 3x3 neighorhoods. In that case, we can eliminate the class `FalsePadded`, create a larger image padded with zeros, and change the loop to `for i=2:h-1, j=2:w-1`. Makes sense?

Reply:
I was trying to avoid relocations. But, yes it is almost the same using the bigger image and assign 
    marker[i-1,j-1]=false 
to delete a pixel